fuzzy-search/info.json

```diff
@@ -57,12 +48,91 @@
                     "visible": true,
                     "editable": true,
                     "value": 75,
-                    "description": "Specify threshold to match key value data in JSON.",
-                    "tooltip": "Specify threshold to match key value data in JSON."
+                    "tooltip": "Specify the similarity threshold (percentage) to match key value data in JSON.",
+                    "description": "Specify the similarity threshold (percentage) to match key value data in JSON."
+                },
+                {
+                    "title": "Fuzzy Match",
+                    "type": "checkbox",
+                    "name": "fuzzyMatch",
+                    "required": false,
+                    "visible": true,
+                    "editable": true,
+                    "value": true,
+                    "tooltip": "Check to filter data for fuzzy format.",
+                    "description": "Check to filter data for fuzzy format.",
+                    "onchange": {
+                        "true": [
+                            {
+                                "name": "cutoff",
+                                "title": "Cut Off",
+                                "description": "Specify Cut Off (percentage) to filter data for Fuzzy Match.",
+                                "type": "integer",
+                                "editable": true,
+                                "visible": true,
+                                "required": true,
+                                "value": 75,
+                                "tooltip": "Specify Cut Off (percentage) to filter data for Fuzzy Match."
+                            }
+                        ]
+                    }
                 }
             ],
             "output_schema": {},
             "open": false
+        },
+        {
+            "title": "Search Keyword In Text",
+            "operation": "search_keyword_in_text",
+            "description": "Searches specified fuzzy keyword in text",
```

Comment:
Suggestion: Fuzzy search specified keyword in an input text

Reply:
Incorporated